terraform/storage.tf

```diff
@@ -1,10 +1,11 @@
 resource "azurerm_storage_account" "main" {
-  name                            = "sacdtcalitp${lower(local.env_letter)}001"
-  location                        = data.azurerm_resource_group.main.location
-  resource_group_name             = data.azurerm_resource_group.main.name
-  account_tier                    = "Standard"
-  account_replication_type        = "RAGRS"
-  allow_nested_items_to_be_public = false
+  name                             = "sacdtcalitp${lower(local.env_letter)}001"
+  location                         = data.azurerm_resource_group.main.location
+  resource_group_name              = data.azurerm_resource_group.main.name
+  account_tier                     = "Standard"
+  account_replication_type         = "RAGRS"
+  allow_nested_items_to_be_public  = false
+  cross_tenant_replication_enabled = true
```

Comment:
Curious why this was set to `true` here? I don't think we need this enabled. In https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/pull/50 we made it `false`.

Reply:
terraform reports the same change whether the prop is omitted or set to `false` but i'll readily admit i didn't think things through any further than that. 😇 

```tf
~ resource "azurerm_storage_account" "main" {
      ~ cross_tenant_replication_enabled   = true -> false
```